pkg/gitreceive/build.go

```diff
@@ -211,17 +206,15 @@ func build(conf *Config, s3Client *storage.Client, kubeClient *client.Client, fs
 	log.Debug("Done")
 
 	log.Debug(
-		"Polling the S3 server every %s for %s for the resultant slug at %s/%s",
+		"Polling the S3 server every %s for %s for the resultant slug at %s",
 		conf.ObjectStorageTickDuration(),
 		conf.ObjectStorageWaitDuration(),
-		conf.Bucket,
 		slugBuilderInfo.AbsoluteSlugObjectKey(),
 	)
 	// poll the s3 server to ensure the slug exists
 	if !usingDockerfile {
 		if err := storage.WaitForObject(
-			s3Client,
-			conf.Bucket,
+			storageDriver,
```

Comment:
Now we are using cli libraries to access object storage do we have to do this check 
@arschles @kmala